src/components/ui4/scenes/HomeSceneUi4.tsx

```diff
@@ -71,6 +76,61 @@ export const HomeSceneUi4 = (props: Props) => {
       .catch(e => console.log(String(e)))
   }, [])
 
+  // Check for PromoCard2 from info server:
+  React.useEffect(() => {
+    fetchInfo(`v1/promoCards2/${config.appId ?? 'edge'}`)
```

Comment:
Use the `v2/rollup` endpoint which gives you EVERYTHING on the info server. Ideally this should be queried at app boot time and stored in redux for use by any component, but for now, query it in one useEffect and use it for both blogPosts and promocard2

Reply:
There's a bug in the `v1/infoRollup` endpoint. It does not properly clean the `osVersion` because it's using `asSemver`.

It should be using a string for android (`"api33"`) and a number for iOS. This will allow a distinction between the two in case somehow in the future there is an overlap.

Created a task:
 https://app.asana.com/0/1200382638405084/1206315466361651/f

Reply:
We shouldn't be using the Android API version, that's not granular enough as there can be regressions within releases with the same API version. See this chart

https://en.wikipedia.org/wiki/Android_version_history

We should be using the actual android version number which is semver based

Reply:
This is not a bug, it's a feature. We should be using the true Android version which is semver based, not the API level. There can be regressions in versions within the same API level so the API level isn't granular enough.

```
import DeviceInfo from 'react-native-device-info';

// Get the Android version
const androidVersion = DeviceInfo.getSystemVersion();
```

Reply:
See this chart https://en.wikipedia.org/wiki/Android_version_history